src/main/java/com/boot/cart/controller/CartController.java

```diff
@@ -42,7 +42,7 @@ public ResponseEntity<CartDTO> addProductToCart(@Email(message = "Invalid email!
         return new ResponseEntity<>(newCart, HttpStatus.CREATED);
     }
 
-    @PutMapping("/updateProductToCart/{email}/{productName}/{quantity}")
+    @PutMapping("/update/{email}/{productName}/{quantity}")
```

Comment:
if you pass the productName and quantity in the RequestBody you could use a PUT /product/{email}/{productName} and for the previous endpoint POST  /product/{email}

Reply:
same as for the first remark.